qa/tasks/scripts/ses_disk_fault_injection.sh

```diff
@@ -9,8 +9,12 @@ function health_ok() {
     done
 }
 
-storage_device_name=$(salt $disk_storage_minion cmd.run \
- "pvdisplay | grep -B 1 'VG Name .* ceph' | egrep -v 'ceph-block-dbs|nvme|--' | head -1 | cut -d / -f 3" --output=json | jq -r .[])
+minion_host="$(cut -d '.' -f1 <<< $disk_storage_minion)"
+random_osd=$(ceph osd tree | grep -A 1 $minion_host | grep -o "osd\.".* | awk '{print$1}')
+osd_id="$(cut -d '.' -f 2 <<< $random_osd)" # osd.0 -> 0
+disk_path=$(salt \* cephdisks.find_by_osd_id ${osd_id} --out json 2> /dev/null | jq -j '.[][].path')
```

Comment:
is it possible to get disk path of the osd from python?

Reply:
Maybe @smithfarm has any clue?

Reply:
something for a different PR if this one turns out to be insufficient